src/test/java/teammates/test/cases/ui/browsertests/AdminSearchPageUiTest.java

```diff
@@ -74,6 +78,10 @@ private void testSearch() {
         assertTrue(isSearchPanelPresent());
         assertTrue(isSearchDataDisplayCorrect());
         
+        StudentAttributes student = testData.students.get("student1InCourse1");
+        InstructorAttributes instructor = testData.instructors.get("instructor1OfCourse1");
```

Comment:
This test data might be incorrect. Could you check if that is the case? The instructor with the 'Manager' role is actually given the permissions for a 'Co-owner'